cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/LogHandlers.java

```diff
@@ -1043,7 +1095,7 @@ public static void putLogLevel(HttpServletRequest request, HttpServletResponse r
         
         try {
             CommonConstants.LoggingLevels.valueOf(logLevel.toUpperCase());
-            LogManager.getRootLogger().setLevel(Level.toLevel(logLevel.toUpperCase()));
+            setLevel(LogManager.getRootLogger(), Level.toLevel(logLevel.toUpperCase()));
```

Comment:
This should be updated having in account: https://github.com/telefonicaid/fiware-cygnus/pull/2170